credentials/bc-petroleum-and-natural-gas-title/governance.md

```diff
@@ -132,9 +132,118 @@ The primary use of the Petroleum and Natural Gas Title credential is for credent
+
+#### Lease
+A subsisting lease issued under this Act
+
+#### Leasee
+A person in whose name a lease is recorded in the division records
+
+#### Licence
+A drilling licence or a licence under section 126 (3) or 130 (3)
+
+#### Licensee
+A person in whose name a licence is recorded in the division records
+
+#### NaturalGas
+All fluid hydrocarbons, before and after processing, that are not defined as petroleum, and any of the following that is produced from a well:
+carbon dioxide; helium; hydrogen; hydrogen sulphide;
+
+#### Permit
+A subsisting exploration permit issued under this Act
+
+#### Permitee
+The person in whose name an exploration permit is recorded in the division records
+
+#### Petroleum
+Crude petroleum and all other hydrocarbons, regardless of gravity, that are or can be recovered in liquid form from a pool through a well by ordinary production methods or that are or can be recovered from oil sand or oil shale
+
+#### PetroleumWell
+A well in which casing is run and that, in the opinion of the regulator, is producing or is capable of producing from a petroleum bearing zone
+
+#### Well
+A hole in the ground
+(a) made or being made by drilling, boring or any other method to obtain petroleum or natural gas,
+(b) made or being made by drilling, boring or any other method to explore for, develop or use a storage reservoir for the storage or disposal of petroleum, natural gas, carbon dioxide, water produced in relation to the production of petroleum or natural gas, waste or any other prescribed substance,
+(c) used, drilled or being drilled to inject natural gas, water produced in relation to the production of petroleum or natural gas or other substances into an underground formation in connection with the production of petroleum or natural gas,
+(d) used to dispose of petroleum, natural gas, water produced in relation to the production of petroleum or natural gas, waste or any other prescribed substance into a storage reservoir, or
+(e) used, drilled or being drilled to obtain geological or geophysical information respecting petroleum or natural gas,
+
+#### Zone
+A stratum or strata designated by the regulator as a zone generally or for a designated area or a specific well, or a stratum or strata designated by the director for the purposes of the administration of petroleum and natural gas rights or storage and disposal rights.
+
+#### Tract
+A Tract is an area of land within a title defined by locations sharing identical rights. This attribute has all of the tract(s) information contained within a title. See Tracts Appendix below for a detailed description of the information included.
+
+__attributes__ 
```

Comment:
Change heading type to be more distinct - and give some explanation of type versus attribute